src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1429,6 +1441,21 @@ protected MembersAndInitializers GetMembersAndInitializers()
             return membersAndInitializers!;
         }
 
+        [Conditional("DEBUG")]
+        internal void AssertMemberExposure(Symbol member)
+        {
+            var declared = _lazyDeclaredMembersAndInitializers;
+            Debug.Assert(declared != DeclaredMembersAndInitializers.UninitializedSentinel);
+
+            if ((declared is object && (declared.NonTypeMembers.Contains(m => m == (object)member) || declared.RecordPrimaryConstructor == (object)member)) ||
```

Comment:
>(declared.NonTypeMembers.Contains(m => m == (object)member) || declared.RecordPrimaryConstructor == (object)member)) || [](start = 39, length = 119)

Nit: considering putting this on the next line to make it clear where the parenthesized expression starts and ends.

Reply:
Actually, I misread. This is fine.

---
In reply to: [568850528](https://github.com/dotnet/roslyn/pull/50961#discussion_r568850528) [](ancestors = 568850528)